hexa/workspaces/migrations/0007_alter_workspace_slug.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.1.3 on 2023-02-06 13:33
+
+import re
+
+import django.core.validators
+from django.db import migrations, models
+
+
+def migrate_workspace_slug(apps, schema_editor):
+    from hexa.workspaces.models import Workspace, create_workspace_slug
```

Comment:
You should use `apps.get_model` to prevent issues with model versions

Reply:
Ok and what about the create_workspace_slug ? Can we leave it like this or is there another way to import it ? @pvanliefland

Reply:
For create_workspace_slug it's alright